### PR TITLE
Issue 3 user can edit an entry

### PR DESCRIPTION
**Pull request by OSAdams**
https://imgur.com/rGnLel5

File too big for GitHub :(

### CODE REVIEW

File: js/main.js [hunk 1]
```diff
@@ -17,24 +18,55 @@ $inputForm.addEventListener('input', event => {
...
 
 $viewEntries.addEventListener('click', viewEntries);
 
 $viewForm.addEventListener('click', viewForm);
 
+$entryList.addEventListener('click', event => {
+  if (event.target.className === 'fas fa-edit entry-edit') {
+    const dataEntryId = parseInt(event.target.getAttribute('data-entry-id'));
+    viewForm();
+    for (const entryIndex in data.entries) {
```
Comment (BrettAlbright) [inline js/main.js]: I would use a for of loop here

**Review by BrettAlbright — CHANGES_REQUESTED** [review]
tiny update!

**Comment by OSAdams** [comment]
code-journal-issue-3-2

**Review by BrettAlbright — APPROVED** [review]
👍🏼

**Pull request merged**